.gitignore

```diff
@@ -180,3 +180,6 @@ project.lock.json
 .vs/
 Tests/
 TestFiles/
+DocumentFormat.OpenXml.Tests/Spreadsheet.xltx
```

Comment:
I don't see any new tests using these. Is that expected? Or did you not add these files and the tests?